tests/utils/command_line_params.cpp

```diff
@@ -1020,6 +1032,47 @@ void add_su3_option_group(std::shared_ptr<QUDAApp> quda_app)
                       "Measure the field energy and topological charge every Nth step (default 5) ");
 }
 
+void add_heatbath_option_group(std::shared_ptr<QUDAApp> quda_app)
```

Comment:
move these to the tests they are used in ?

Reply:
1e742aec4